source/reference/beta-program/abc-migration.txt

```diff
@@ -52,6 +52,13 @@ before the second migration (B->C) starts committing.
    review the :ref:`{+c2c-full-beta-program+} Disclaimer 
```

Comment:
Above at the top of behavior, we also want to say something like: "For best performance, the first migration (A->B) should reach change event application before the second migration (B->C) starts." We should also explain how to check if a migration has reached change event application (checking the info field of the progress endpoint: https://preview-mongodbajhuhmdb.gatsbyjs.io/cluster-sync/DOCSP-41581-beta-docs-cleanup/reference/api/progress/#successful-response)

Reply:
Updated.